simulation/tests/test_agents.py

```diff
@@ -538,3 +538,25 @@ def test_total_heating_system_costs_are_lower_for_heat_pumps_if_model_interventi
         assert household.get_total_heating_system_costs(
             heat_pump, model_with_rhi
         ) < household.get_total_heating_system_costs(heat_pump, model_without_rhi)
+
+    @pytest.mark.parametrize("heating_system", list(HeatingSystem))
+    def test_heating_fuel_costs_appear_lower_for_landlords_than_owner_occupiers(
```

Comment:
I think the test should be a more specific: `test_heating_fuel_costs_are_zero_for_landlords`.